src/test/scala/cognite/spark/v1/RawTableRelationTest.scala

```diff
@@ -894,4 +909,40 @@ class RawTableRelationTest
     err.getMessage shouldBe "Error while loading RAW row [key='k'] in column 'value': java.lang.NumberFormatException: For input string: \"test\""
 
   }
+
+  it should "filter out fields with null value but not impact schema inference" in {
```

Comment:
let's also add test for no-null-filtering case

Reply:
As if it isn't enabled, it should behave as before? Sure, can do that.

Reply:
Ah, I think what I actually wanted is no-inference case with null filtering and without, both outcomes of there being a diff and no diff will be interesting.

For inference case not a lot to see looking at resulting dataframe, but otoh extra small test never hurts